pkg/proxy/ipvs/ipset.go

```diff
@@ -78,6 +78,9 @@ const (
 
 	kubeHealthCheckNodePortSetComment = "Kubernetes health check node port"
 	kubeHealthCheckNodePortSet        = "KUBE-HEALTH-CHECK-NODE-PORT"
+
+	kubeIpvs0SetComment = "Addresses on interface kube-ipvs0"
+	kubeIpvs0Set        = "KUBE-IPVS0-IPS"
```

Comment:
FWIW, `"kube-ipvs0"` is currently defined as a constant (`defaultDummyDevice`) in `proxier.go`, and the name kind of implies that the dummy device _could_ be something "non-default" instead, although in fact, there's no support for actually doing that. So anyway, if you're going to start explicitly referencing the interface name elsewhere, maybe `defaultDummyDevice` should be renamed to something that makes it clearer that it's not just a "default"? (in a separate commit)